hack/update-generated-kubelet-plugin-registration-dockerized.sh

```diff
@@ -23,13 +23,17 @@ set -o nounset
 set -o pipefail
 
 KUBE_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../" && pwd -P)"
-KUBELET_PLUGIN_REGISTRATION_V1ALPHA="${KUBE_ROOT}/staging/src/k8s.io/kubelet/pkg/apis/pluginregistration/v1alpha1/"
-KUBELET_PLUGIN_REGISTRATION_V1BETA="${KUBE_ROOT}/staging/src/k8s.io/kubelet/pkg/apis/pluginregistration/v1beta1/"
-KUBELET_EXAMPLE_PLUGIN_V1BETA1="${KUBE_ROOT}/pkg/kubelet/pluginmanager/pluginwatcher/example_plugin_apis/v1beta1/"
-KUBELET_EXAMPLE_PLUGIN_V1BETA2="${KUBE_ROOT}/pkg/kubelet/pluginmanager/pluginwatcher/example_plugin_apis/v1beta2/"
+
+dirs=(
+    "staging/src/k8s.io/kubelet/pkg/apis/pluginregistration/v1alpha1/"
+    "staging/src/k8s.io/kubelet/pkg/apis/pluginregistration/v1beta1/"
+    "staging/src/k8s.io/kubelet/pkg/apis/pluginregistration/v1/"
```

Comment:
is it possible to make this list directories under `staging/src/k8s.io/kubelet/pkg/apis/pluginregistration/v*` or something so we pick up all versioned subdirs, including any future ones? same question for example_plugin_apis and podresources below

Reply:
yes, I want to followup and not make this a hardcoded list.

Also, we have like 5 different scripts which all run the same command on different dirs - maybe just one script (update-codegen) ?